docs/content/faq/how-do-i-use-the-postgres-appliance.md

```diff
@@ -0,0 +1,18 @@
+---
+title: How do I use the Postgres appliance?
+layout: docs
+toc_min_level: 2
+---
+
+# How do I use the Postgres appliance?
```

Comment:
This seems redundant with https://github.com/flynn/flynn/blob/master/docs/content/postgres.html.md